dm-tool: add page (#7691)
tldr-pages/tldr: pages/linux/dm-tool.md

> # dm-tool

Reviewer: Need an empty line.

> A tool to commnicate with the lightdm display manager. EG: lock currect session and show lightdm, list all open sessions, connect an existing x server to the display manager.

Reviewer: ```suggestion
> A tool to control the display manager.
```

> More informatioin: https://www.mankier.com/1/dm-tool.

Reviewer: ```suggestion
> More information: <https://www.mankier.com/1/dm-tool>.
```

> - show the greeter while keeping current desktop session in ram and waiting to be restored upon autentication by logged in uuser
> `dm-tool switch-to-greeter`

Reviewer 2: ```suggestion
- Show the greeter while keeping current desktop session in ram and waiting to be restored upon authentication by logged in user:

`dm-tool switch-to-greeter`
```

> - swicth user, showing a authentication prompt if required
> `dm-tool swict-to-user {{username}} [{{session}}]`

Reviewer 2: ```suggestion
- Switch to the specified user, showing an authentication prompt if required:

`dm-tool switch-to-user {{username}} [{{session}}]`
```

> - lock the current session
> `dm-tool lock`

Reviewer 2: ```suggestion
- Lock the current session:

`dm-tool lock`
```

> - add a new X server instance
> `dm-tool add-seat [{{name}}={{value}}]`

Reviewer 2: ```suggestion
- Add a new X server instance:

`dm-tool add-seat [{{name}}={{value}}]`
```
